.gitlab/functional_test/security_agent.yml

```diff
@@ -17,6 +17,10 @@
     - tasks/run-test-kitchen.sh security-agent-test $AGENT_MAJOR_VERSION
     - popd
     - inv system-probe.print-failed-tests --output-dir $DD_AGENT_TESTING_DIR/testjson
+  after_script:
+    # Upload generated junit files
+    - export DATADOG_API_KEY=$($CI_PROJECT_DIR/tools/ci/aws_ssm_get_wrapper.sh ci.datadog-agent.datadog_api_key_org2)
+    - ss=0; for f in $DD_AGENT_TESTING_DIR/kitchen-junit-*.tar.gz; do [[ -e "$f" ]] || continue; inv -e junit-upload --tgz-path $f || ((ss++)); done; exit $ss
```

Comment:
This block is checking for upload errors, while the previous don't seem to.

Is that on purpose?

Do you think you could move the duplicated `after_script` to some common code?

Reply:
As discussed, ACIX-191

Reply:
As discussed with Nicolas, a follow up task has been opened to handle this later